### DISCUSSION TITLE
Pagination support

### DISCUSSION THREAD

**Discussion by franq77** [grafana/grafana-json-datasource#74] [Q&A] [answered]
Love this plugin! 

I am just wondering if it can do pagination? 
I connect to a third party and it only returns me 500 items. Should I need more I have send another query with an offset of 501. 
What would you advice to solve this?

Regards,
Frank

**Comment by marcusolsson** [comment, marked as answer]
Unfortunately not. This has been requested previously and I'd love to be able to support it. But since there's many ways to implement pagination, I haven't yet found a good way to implement it generically.

For now, I'd recommend using dashboard variables. I've even been thinking about building a pagination panel that syncs with set of dashboard variables that you could then use in your query.
